test/multipoint_routes.cc

```diff
@@ -74,6 +74,9 @@ struct route_tester {
     loki_worker.route(request);
     thor_worker.route(request);
     odin_worker.narrate(request);
+    loki_worker.cleanup();
+    thor_worker.cleanup();
+    odin_worker.cleanup();
```

Comment:
when using this example for my other test i realized i forgot to do this when i made this test so i cleaned it up. if you dont call these then some state is kept between requests. we dont reuse this object ever in the tests below so it doesnt matter but whatever. future someone who is writing tests will thank me